lib/cocoapods/downloader/cache.rb

```diff
@@ -0,0 +1,118 @@
+require 'fileutils'
+require 'tmpdir'
+
+module Pod
+  module Downloader
+    class Cache
+      Result = Struct.new(:location, :spec, :checkout_options)
+
+      attr_reader :root
+
+      def initialize(root)
+        @root = root
+        root.mkpath unless root.exist?
+      end
+
+      def download_pod(request)
+        cached_pod(request) || uncached_pod(request)
+      rescue Informative
+        raise
+      rescue
+        UI.notice("Error installing #{request.name}")
```

Comment:
Maybe we can provide a bit more information in this error (perhaps whatever the exception is?).

As a user if I see `Error installing RestKit` I'd be lost and have no idea where to being making it work again. Would be nice to at least have some kind of hint.